pipelinewise/fastsync/commons/tap_postgres.py

```diff
@@ -350,6 +350,10 @@ def fetch_current_incremental_key_pos(self, table, replication_key):
         elif isinstance(postgres_key_value, decimal.Decimal):
             key_value = float(postgres_key_value)
 
+        if key_value is None:
+            LOGGER.warning('No replication value found for table %s, returning empty bookmark', table)
+            return {}
+
```

Comment:
It is better to move these lines between lines 341 and 344, exactly after getting 'key_value'. because in a case it is None, then it should pass `IF` clauses in lines 344, 347 and 350 and none of those if cases will be true. so by moving 'if key_value is None' before them, we will save some processing time.

Reply:
addressed